util/steambox

```diff
@@ -51,9 +51,12 @@ declare -a DRIDEVS=()
 if [[ -d /dev/dri ]] ; then
 	DRIDEVS+=(/dev/dri/*)
 fi
-if [[ -d /dev/ati ]] ; then 
+if [[ -d /dev/ati ]] ; then
 	DRIDEVS+=(/dev/ati/*)
 fi
+if [[ -e /dev/nvidiactl ]] ; then
+	DRIDEVS+=(/dev/nvidia*)
```

Comment:
@rsharo The glob was intentional, but I've made a few assumptions.  Here are the devices nodes that the nvidia kmods are creating on my system:
```
crw-rw-rw- 1 root root 195,   0 Jan 22 20:26 /dev/nvidia0
crw-rw-rw- 1 root root 195, 255 Jan 22 20:26 /dev/nvidiactl
crw-rw-rw- 1 root root 195, 254 Jan 22 20:26 /dev/nvidia-modeset
```
I chose to test for `/dev/nvidiactl` as it had the highest minor device number and the name felt stable.  *Dubious reasoning on my part, at best!*  

My hope was that the glob would do a passable job of including *likely* device nodes that might employed by other releases of the driver, other card models, or systems with multiple display adapters.  It's perhaps better to only include the specific device nodes I know to be necessary.